From 041323dfee5a06166ca37d59e8108229ab7973d1 Mon Sep 17 00:00:00 2001
From: Brahma Reddy Battula <brahma@apache.org>
Date: Mon, 26 Sep 2016 22:31:31 +0530
Subject: [PATCH 2047/2748] HADOOP-12597. In kms-site.xml configuration
 hadoop.security.keystore.JavaKeyStoreProvider.password
 should be updated with new name. ( Contributed by
 Surendra Singh Lilhore via Brahma Reddy Battula)

(cherry picked from commit aaff2033b92863a92c7e206f5eb4319b35e713d0)

 Conflicts:
	hadoop-common-project/hadoop-common/CHANGES.txt

Change-Id: I12325ee8c1a74ac11e864fa12b5cb481638fb163
---
 .../hadoop-kms/src/main/conf/kms-site.xml          |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/hadoop-common-project/hadoop-kms/src/main/conf/kms-site.xml b/hadoop-common-project/hadoop-kms/src/main/conf/kms-site.xml
index 51b4438..6c8becf 100644
--- a/hadoop-common-project/hadoop-kms/src/main/conf/kms-site.xml
+++ b/hadoop-common-project/hadoop-kms/src/main/conf/kms-site.xml
@@ -25,10 +25,10 @@
   </property>
 
   <property>
-    <name>hadoop.security.keystore.JavaKeyStoreProvider.password</name>
-    <value>none</value>
+    <name>hadoop.security.keystore.java-keystore-provider.password-file</name>
+    <value>kms.keystore.password</value>
     <description>
-      If using the JavaKeyStoreProvider, the password for the keystore file.
+      If using the JavaKeyStoreProvider, the file name for the keystore password.
     </description>
   </property>
 
-- 
1.7.9.5

